content/en/user-guide/chaos-engineering/_index.md

```diff
@@ -3,27 +3,25 @@ title: "Chaos Engineering"
 linkTitle: "Chaos Engineering"
 weight: 11
 description: >
-  Chaos Engineering with LocalStack enables you to build resilient systems early on in the development phase.
+  Chaos Engineering with LocalStack enables you to build resilient systems early on
```

Comment:
the ending of the sentence somehow got lost.

Reply:
Fixed in d0dcd3faa